etc/behat/services/contexts/ui.xml

```diff
@@ -29,38 +29,38 @@
     <services>
         <service id="sylius.behat.context.ui.checkout" class="%sylius.behat.context.ui.checkout.class%" scope="scenario">
             <argument type="service" id="sylius.behat.shared_storage" container="symfony" />
-            <argument type="service" id="sylius.behat.page.checkout_security" />
-            <argument type="service" id="sylius.behat.page.checkout_addressing" />
-            <argument type="service" id="sylius.behat.page.checkout_shipping" />
-            <argument type="service" id="sylius.behat.page.checkout_payment" />
-            <argument type="service" id="sylius.behat.page.checkout_finalize" />
-            <argument type="service" id="sylius.behat.page.checkout_thank_you" />
-            <argument type="service" id="sylius.behat.page.order_payments" />
+            <argument type="service" id="sylius.behat.page.shop.checkout_security" />
+            <argument type="service" id="sylius.behat.page.shop.checkout_addressing" />
+            <argument type="service" id="sylius.behat.page.shop.checkout_shipping" />
```

Comment:
WDYT about `sylius.behat.page.shop. checkout.shipping`? It is not `CheckoutShippingStep`. we have `ShippingStep` in checkout folder